README.md

```diff
@@ -133,6 +133,17 @@ It supports the popular IDEs for managing GraphQL requests and exploring GraphQL
 - [GraphQL Playground](https://github.com/prisma-labs/graphql-playground)
 - [Voyager](https://github.com/APIs-guru/graphql-voyager)
 
+## Ahead-of-time compilation
+
+GraphQL.NET supports ahead-of-time (AOT) compilation for execution of code-first schemas with .NET 7. This allows for use within
+iOS and Android apps, as well as other environments where JIT compilation is not available. It may be necessary to explicitly
```

Comment:
```suggestion
iOS and Android apps, as well as other environments where such features as JIT compilation or
dynamic code generation are not available. It may be necessary to explicitly
```